### ISSUE TITLE
User request for FAIR data auto access request

### ISSUE THREAD

**Issue by sgworth** [irods-contrib/metalnx-web#49]
Request is to provide an automation mechanism in Metalnx such that when a search request provides a link to an object the user does not have access to a permissions request is automatically generated to the owner of the data (user or group) to let that owner grant access if appropriate.

**Comment by michael-conway** [comment]
Initial work in progress is being done in the NIEHS fork, we can merge this as part of the 2.1.0? Added to that milestone so we can look at that....

**Comment by michael-conway** [comment]
example asof version 2.0.1....here a screen shows access to a file that the user does not have permission to see...shows basic data and avu metadata...

![access](https://user-images.githubusercontent.com/1097261/42048264-bc369204-7ad0-11e8-8956-e1b7333277b2.png)


results now in email...very basic....


<img width="1111" alt="mail" src="https://user-images.githubusercontent.com/1097261/42048287-d6c8ae40-7ad0-11e8-8b56-e0599f17f5f5.png">

**Comment by michael-conway** [comment]
metalnx.properties value access.proxy must be = true to get this page, otherwise it's a not found page...


mail.enabled must be true to get the 'request access' button, otherwise it's not shown. 

So this whole system can be turned off in metalnx.properties if FAIR access is not required.

**Comment by trel** [comment]
mail template and mail from address configured separately?

**Comment by michael-conway** [comment]
Yes.  Right now it can have a mail.from and a mail.to, if the to is not
specified it uses the (new) admin email id. All that is in the
/etc/irods-ext/metalnx.properties and shown in the example template.

**Comment by michael-conway** [comment]
see now in irods-contrib 'dev' branch for example...

https://github.com/irods-contrib/metalnx-web/blob/development/etc/irods-ext/metalnx.properties#L105

**Comment by trel** [comment]
I see.

A template for the email body?

Plaintext?  HTML?    Just asking - I have no agenda.

**Comment by michael-conway** [comment]
html would be nice and is generally contemplated, right now we're in
expediency mode

**Comment by michael-conway** [comment]
this is there...just need some cleanups and likely should tie into notifications system so let's kick to 2.2.1

**Comment by trel** [comment]
metalnx will not be sending any notifications.

closing.
